### ISSUE TITLE
[svelte-parser] IfBlock is an invalid element : the mapping node is broken (parse-error)

### ISSUE THREAD

**Issue by shunito** [markuplint/markuplint#698]
- Markuplint version: `3.2.0`
- Parser lang: svelte-parser
- Node.js version: `v18.12.1`
- OS: macOS

## Describe the bug
- The output is "IfBlock is an invalid element : the mapping node is broken (parse-error)".
- However, the syntax looks fine, and Svelte can Build.

![VS Code screenshot, line368 is error ](https://user-images.githubusercontent.com/32133/217251199-97d4cc19-2a98-409b-b8aa-b3d329807d8a.png)

## Code Example

```html
      {#if Math.abs(contrastLc) >= 90}<li>
          WCAG 2.1の達成基準 1.4.6 コントラスト (高度) 7:1（AAA）以上と同等。
        </li>
      {:else if Math.abs(contrastLc) >= 75}<li>
          WCAG 2.1の達成基準 1.4.3 コントラスト (最低限) 4.5:1（AA）以上と同等。
        </li>
      {:else if Math.abs(contrastLc) >= 60}<li>
          WCAG 2.1の達成基準 1.4.3 コントラスト (最低限)
          3:1（大きな文字の場合）（AA）以上と同等。
        </li>
      {/if}
```

## Steps To Reproduce

1.  Clone repository https://github.com/shunito/apca-calculator-ja.
2.  Run `markuplint './src/**/*.svelte'`

## The current behavior
- The last line `{/if}`, results in an error. 

## The expected behavior
- The last line `{/if}`, results not in an error.
